server/client.go

```diff
@@ -252,6 +255,18 @@ type AutoCompleteResult struct {
 	Results []Result `json:"results"`
 }
 
+type Item struct {
```

Comment:
What's Item? We can find a more relevant name I think

Reply:
Now I have changed this to `GroupItem`. It indicates the individual group.